### DISCUSSION TITLE
decodeXLHeaders: Unknown xl meta version 2

### DISCUSSION THREAD

**Discussion by axe4man** [minio/minio#16393] [Q&A]
Hi. Today I got such a problem on all 3 nodes minio. I can't understand the reason and the method of recovery.

Error: Drive /data2, path (.minio.sys/pool.bin) returned an error (decodeXLHeaders: Unknown xl meta version 2) (*fmt.wrapError)

**Comment by klauspost** [comment]
Seems like you have downgraded. That is not supported.

**Comment by axe4man** [comment]
As a result, we installed a new version, and it helped. But. No one has carried out any work with minio since the beginning of the year. And there are no traces in the logs.

**Reply by klauspost** [reply]
Then something is going on. Data written from a newer version isn't going to appear out of the blue.

**Comment by ell1e** [comment]
I also just ran into this, despite the minio version being pinned. Seems like a bug?
